golog.go

```diff
@@ -128,6 +131,11 @@ type Logger interface {
 	// Debugf logs to stdout
 	Debugf(message string, args ...interface{})
 
+	// Debug logs to stdout
```

Comment:
Is this Debug or Info? FWIW, when I wrote golog I consciously chose to avoid the INFO level because I've found that a proliferation of log levels leads to confusion over what should be logged at what level.

Reply:
Yeah I hear you. Overall zap and similar seem to have the same number of levels (basically info and no trace).